tests/test_core.py

```diff
@@ -0,0 +1,15 @@
+import json
+import tempfile
+from dlh_ingest import ingest_from_config
```

Comment:
does this test work for you? package is named `data_lakehouse_ingest` in pyproject.toml but here you are import from `dlh_ingest`